Level0/Hello_World/README.md

```diff
@@ -0,0 +1,49 @@
+# Smart Contract "Hello World" in Remix-IDE
+
+Smart contracts are contracts that execute themselves. The Remix is an Integrated Development Environment (IDE) for creating smart contracts in Solidity. The Solidity code can be written, compiled, and debugged using the IDE.
+
+
+## Getting Started :
+
+### Step 1:
+
+For opening the remix-ide in browser type: https://remix.ethereum.org/.
+
+A screen will be presented like the following:
+
+![image](https://user-images.githubusercontent.com/75573569/182060674-215aa896-cf26-41e7-b484-c249c1033ece.png)
+
+Click on the “file explorer” icon onto the left side bar.
+
+Select Solidity in the Environment and click + symbol right to the browser.
+
+Type the file name  and enter the respective code into it.
+
+
+### Step 2:
+Now compile the hello.sol file,
+
+![1](https://user-images.githubusercontent.com/75573569/182083042-e9139c30-7861-45c4-942c-2b3288cad1e0.png)
+
+
+### Step 3:
+Once compiled successfully, click “Deploy and run transactions”. and "Deploy" it ,
+
+![2](https://user-images.githubusercontent.com/75573569/182084425-c20685b8-c98d-4eed-8798-df532f7b5eec.png)
+
+
+### Step 4:
+Here's the ABI and Bytecode -
+
+![3](https://user-images.githubusercontent.com/75573569/182084864-8e904c96-f529-474d-b2ff-e8d096a40db8.png)
+
+And here you can see the contract address - 
+
+![4](https://user-images.githubusercontent.com/75573569/182086187-826c6d31-8018-4e5c-8a0c-844b4c5ed540.png)
+
+### Step 5:
+Click “>” before your contract you will see a button “hello” below as the contract has hello function that returns a string. 
+Click get button and you will get :

```

Comment:
remove 46th line and write this : 

Click  hello button and you will get the output